src/view/components/consentObjects.jsx

```diff
@@ -0,0 +1,253 @@
+                                />
+                              )}
+                            {value.standard === ADOBE.value &&
+                              value.adobeVersion !== VERSION_1_0.value && (
+                                <DataElementSelector>
+                                  <TextField
+                                    data-test-id="valueField"
+                                    name={`consent[${index}].value`}
+                                    label="Value"
+                                    description="Provide a data element containing the Adobe consent XDM object"
+                                    width="size-5000"
+                                    isRequired
+                                  />
+                                </DataElementSelector>
+                              )}
+                            {value.standard === IAB_TCF.value && (
+                              <>
+                                <TextField
+                                  data-test-id="iabVersionField"
+                                  name={`consent[${index}].iabVersion`}
+                                  label="Version"
+                                  description="The IAB TCF standard version"
+                                  width="size-5000"
+                                  isRequired
+                                />
+                                <DataElementSelector>
+                                  <TextField
+                                    data-test-id="iabValueField"
+                                    name={`consent[${index}].iabValue`}
+                                    label="Value"
+                                    description="The encoded IAB TCF consent value"
+                                    width="size-5000"
+                                    isRequired
+                                  />
+                                </DataElementSelector>
+                                <OptionsWithDataElement
+                                  label="GDPR Applies"
+                                  name={`consent[${index}].gdprApplies`}
+                                  dataTestIdPrefix="gdprApplies"
+                                  options={[YES, NO]}
+                                  defaultValue={YES.value}
+                                />
+                                <OptionsWithDataElement
+                                  label="GDPR Contains Personal Data"
+                                  name={`consent[${index}].gdprContainsPersonalData`}
+                                  dataTestIdPrefix="gdprContainsPersonalData"
+                                  options={[YES, NO]}
+                                  defaultValue={NO.value}
+                                />
+                              </>
+                            )}
+                            {consent.length > 1 && (
+                              <div className="u-gapTop">
```

Comment:
Upon further review (we discussed this previously), this `<div>` isn't necessary in order to keep the button from being expanded to 100% width. Without the `div`, the button would typically be expanded to 100% width because it's within a `<Flex direction="column">` and by default flex items are stretched along the cross axis of the flex parent. There are a few ways to solve this:

* Set `alignItems="start"` on the Flex element.
* Set `alignSelf="flex-start"` on the Button element.
* Do what you did here.

Reply:
Thanks for the help. I need to study Flex alignment at some point. When I learned CSS, I still had to support IE 6.